src/app/content/components/Page.spec.tsx

```diff
@@ -54,7 +54,7 @@ jest.mock('../../domUtils', () => ({
 
 const makeEvent = (doc: Document) => {
   const event = doc.createEvent('MouseEvents');
-  event.initEvent('click', true, false);
+  event.initEvent('click', true, true);
```

Comment:
This is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent) and I wonder if we should replace usage of it across tests with the [Event() constructor](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event).